yandex/data_source_yandex_container_repository.go

```diff
@@ -25,6 +25,11 @@ func dataSourceYandexContainerRepository() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+
+			"folder_id": {
+				Type: schema.TypeString,
+				Optional: true,
```

Comment:
Unlike some other resources, I did not mark this field as computed, as actual ContainerRepository from the API does not remember its enclosing folder.